root/edit/details/EditArtist.js

```diff
@@ -0,0 +1,180 @@
+              oldText={display.sort_name?.old ?? ""}
+              split="\s+"
+            />
+          ) : null}
+          {display.comment ? (
+            <Diff
+              label={addColonText(l('Disambiguation'))}
+              newText={display.comment?.new ?? ""}
+              oldText={display.comment?.old ?? ""}
+              split="\s+"
+            />
+          ) : null}
+          {type ? (
+            <FullChangeDiff
+              label={addColonText(l('Type'))}
+              newContent={type.new.name}
+              oldContent={type.old.name}
+            />
+          ) : null}
+          {gender ? (
+            <FullChangeDiff
+              label={addColonText(l('Gender'))}
+              newContent={gender.new?.name ?? ""}
+              oldContent={gender.old?.name ?? ""}
+            />
+          ) : null}
+          {area ? (
+            area.new?.gid === area.old?.gid ? null : (
+              <FullChangeDiff
+                label={addColonText(l('Area'))}
+                newContent={area.new ? <DescriptiveLink entity={area.new} /> : null}
+                oldContent={area.old ? <DescriptiveLink entity={area.old} /> : null}
+              />
+            )) : null}
+          {beginDate ? (
+            <Diff
+              label={artistBeginLabel(displayTypeId)}
+              newText={formatDate(beginDate.new)}
+              oldText={formatDate(beginDate.old)}
+              split="-"
+            />
+          ) : null}
+          {beginArea ? (
+            beginArea.new?.gid === beginArea.old?.gid ? null : (
+              <FullChangeDiff
+                label={addColonText(l('Begin Area'))}
+                newContent={beginArea.new ? <DescriptiveLink entity={beginArea.new} /> : null}
+                oldContent={beginArea.old ? <DescriptiveLink entity={beginArea.old} /> : null}
+              />
+            )
+          ) : null}
+          {endDate ? (
+            <Diff
+              label={artistEndLabel(displayTypeId)}
+              newText={formatDate(endDate.new)}
+              oldText={formatDate(endDate.old)}
+            />
+          ) : null}
+          {endArea ? (
+            endArea.new?.gid === endArea.old?.gid ? null : (
+              <FullChangeDiff
+                label={addColonText(l('End Area'))}
+                newContent={endArea.new ? <DescriptiveLink entity={endArea.new} /> : null}
+                oldContent={endArea.old ? <DescriptiveLink entity={endArea.old} /> : null}
+              />
+            )
+          ) : null}
+          {ended ? (
+            <FullChangeDiff
+              label={addColonText(l('Ended'))}
+              newContent={yesNo(ended.new)}
+              oldContent={yesNo(ended.old)}
+            />
+          ) : null}
+          {display.ipi_codes ? (
+          <Diff
+            label={addColonText(l('IPI codes'))}
+            newText={display.ipi_codes?.new ? commaOnlyListText(display.ipi_codes.new) : ""}
```

Comment:
Is `display.ipi_codes.new` not enough if `display.ipi_codes` is checked above? If flow is being stupid you might want to assign it a variable and see if that helps.